src/main/java/net/aicomp/javachallenge2015/Bookmaker.java

```diff
@@ -34,35 +41,32 @@
 	private static int turn;
 	private static int[][] board = new int[MAP_WIDTH][MAP_WIDTH];
 
-	public static void main(String[] args) throws InterruptedException {
+	private static final String EXEC_COMMAND = "a";
+	private static final String PAUSE_COMMAND = "p";
+	private static final String UNPAUSE_COMMAND = "u";
+
+	public static void main(String[] args) throws InterruptedException,
+			ParseException {
 		// AIの実行コマンドを引数から読み出す
-		String[] execAICommands = new String[PLAYERS_NUM];
-		String[] pauseAICommands = new String[PLAYERS_NUM];
-		String[] unpauseAICommands = new String[PLAYERS_NUM];
-		for (int i = 0; i < PLAYERS_NUM; i++) {
-			execAICommands[i] = "";
-			pauseAICommands[i] = null;
-			unpauseAICommands[i] = null;
+		Options options = new Options()
+				.addOption(EXEC_COMMAND, true, "Commands to execute AI program")
```

Comment:
the command and arguments with double quotation marks to execute AI (e.g. -a "java MyAI")